appveyor.yml

```diff
@@ -5,9 +5,8 @@ environment:
   - TARGET: x86_64-pc-windows-msvc
     ALLOW_PR: 1
   - TARGET: i686-pc-windows-msvc
```

Comment:
Ah ok, that makes sense and no worries! My only worry is that this'll conflict with the other i686 builder for the dist artifacts being produced. Enabling the MSI installer just builds more artifacts, right? It shouldn't change the existing binaries?

(if so we can probably just gate on the msi build)